src/GLua/AG/PrettyPrint.ag

```diff
@@ -182,16 +186,26 @@ infixl 6 <->
 a <-> b | a == empty = b
         | b == empty = a
         | otherwise  = a <> zchr ' ' <> b
+
+-- Always 10, so there is a default for precedence
+infixr 5 $$$
+($$$) :: Int -> Int -> Int
+a $$$ b = if a > b then 10 else 10
```

Comment:
The function always returns 10, so it does effectively nothing.

Reply:
Yeah this was supposed to make it so if a node doesn't define precedence, it gets set back to 10. I didn't want to have to define it on other types of nodes. It's hacky.